cmd/minikube/cmd/start.go

```diff
@@ -174,7 +173,7 @@ func init() {
 	provision.SetDetector(&provisioner.MinishiftProvisionerDetector{Delegate: provision.StandardDetector{}})
 }
 
-// initClusterUpFlags creates the CLI flags which needs to be passed on to 'oc cluster up'
+// initStartFlags creates the CLI flags which needs to be passed on to 'minishift start'
```

Comment:
No, so these flags are for libmachine and passed during VM creation. I would've put it like `initStartFlags creates the CLI flags which needs to be passed on to libmachine`